### ISSUE TITLE
Pie Chart and Line Chart not showing on Recent Form Submissions widget

### ISSUE THREAD

**Issue by bhashkar007** [verbb/formie#1605]
### Describe the bug

I have issue when adding "Recent Form Submissions" widget from Craft CMS dashboard by having setting of "Display Type" as Pie Chart or Line Chart. In both cases, chart isn't showing up. I saw the browser's console and it's showing error as below:

```
Uncaught ReferenceError: moment is not defined
    at s.constructor.init (formie-widgets.js:304:9)
    at s.constructor.constructor (Base.js:20:15)
    at s.constructor (Base.js:32:21)
    at new s.constructor (Base.js:32:21)
    at HTMLDocument.<anonymous> (dashboard:684:1)
    at l (jquery.js?v=1697807469:2:30361)
    at c (jquery.js?v=1697807469:2:30663)
```
   

### Steps to reproduce

1. Login to Craft admin dashboard. From "New Widget", add "Recent Form Submissions"
2. In "Recent Form Submissions Settings", select "Display Type" setting as Pie Chart or Line Chart.
3. Save this setting.
4. You will see that "Recent Form Submissions" widget isn't showing chart.

### Form settings

- Multi-page form: No
- Submission Method: Both
- Client-side Validation: Yes
- Custom Form Templates: Yes

### Craft CMS version

4.5.7

### Plugin version

2.0.38

### Multi-site?

No

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-4 as 2.0.38"`.

**Comment by engram-design** [comment]
Fixed in [2.0.39](https://github.com/verbb/formie/releases/tag/2.0.39)
